### ISSUE TITLE
Triple professions bonus

### ISSUE THREAD

**Issue by moonsado** [dalaranwow/dalaran-wow#3061]
I have noticed some players are able to take JC and BS. Create extra slots thanks to BS put in JC epic gems, than they can quit BS or JC and take engineering basically giving a player extra gem sockets + epic JC gems + rocket gloves. Normally you can only have a bonus of 2 professions

**Comment by Drawsblood** [comment]
You lose the bonus' when you drop it though... lmao...

**Comment by moonsado** [comment]
Nope.

**Comment by moonsado** [comment]
For example this warrior has the rockets from engineering + the extra gem slot from BS + the epic gems from JC. I do not know if this is a bug or normal or bug abusing etc.

http://postimg.org/image/iwsicc2bn/

**Comment by Tarpor** [comment]
As you can see it only states that the gem requires 350 JC, not the socket itself. I believe this is only a problem with the Blacksmithing bonus, as the other professions all have restrictions.

**Comment by strobe-light** [comment]
Hi, so I've actually dropped Blacksmithing for Engineering a while ago, so I can show some proof that these sockets are purely visual once you drop BS, there are no stat gains granted anymore. These screenshots are done in Holy spec without BoK, so the only %based stat boosts I have currently are +10% intellect from talents.
http://imgur.com/a/eJDlI
With the first screenshot, I have 239 strength and 1898 intellect. In the 2nd screenshot I remove the bracers (Wristbands of the Sentinel Huntress) which has 56 strength with a 20 strength BS socket, and my strength drops down to 183.  239 - 183 = 56, which is the amount of strength on the bracers without the BS socket included.
I did the same thing with the pharos gloves (screenshots 3&4), 1898 int before and 1744 int after removing them. 1898 - 1744 = 154 int and 154 = (72+34+34)*1.1. Again, I don't receive the bonus 20 intellect from the (now no longer functioning) blacksmithing socket.
tl;dr: After dropping BS, any wrist/glove prismatic sockets added by BS will cease to provide the benefits of any gems within them, so I don't believe there is any bug/exploit involved.
